### PR TITLE
Emerging Properties in Self-Supervised Vision Transformers

### PR DESCRIPTION
**Pull request by nathanpainchaud**

### CODE REVIEW

File: article/_posts/2021-08-20-EmergingPropertiesSSViT.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+
+# Data
+The authors mainly used DINO to pretrain ViT using ImageNet, but in some experiments' setup other (unsupervised)
+datasets were used.
+
+
+# Results
+The full paper (and even more so with the supplementary materials) provides a lot of results on different downstream
+tasks. They also provide a comprehensive ablation study regarding DINO's components, as well as
```
[the remark this answers is not in the document]
Comment (jhlegarreta) [reply article/_posts/2021-08-20-EmergingPropertiesSSViT.md:66]: @nathanpainchaud The last sentence seems unfinished.

**Review by jhlegarreta — COMMENTED** [review]
Sorry for the delay.

Thanks for the text.

File: article/_posts/2021-08-20-EmergingPropertiesSSViT.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nathanpainchaud) [reply article/_posts/2021-08-20-EmergingPropertiesSSViT.md]: Oups :stuck_out_tongue: I didn't understand what you were getting at Carl. Sorry! :sweat_smile:

[the remark this answers is not in the document]
Comment (jhlegarreta) [reply article/_posts/2021-08-20-EmergingPropertiesSSViT.md]: Done in cb2b9ac.

**Review by jhlegarreta — APPROVED** [review]

**Pull request merged**